src/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
+}
```

Comment:
Missing newline.

Reply:
I've actually since removed this file from `src` since it was preventing unit tests from functioning correctly. In order for `proxyquire` to continue functioning, I need everything to run through Babel's CommonJS transform for unit tests (I've accomplished this up by setting `NODE_ENV=test` in `mocharc.js` and setting up a test-specific `.bablerc` config block).